components/health_controller/src/reporters.rs

```diff
@@ -76,10 +90,17 @@ impl UnifiedSlowScore {
         &mut self,
         id: u64,
         factor: InspectFactor,
-        duration: &RaftstoreDuration,
+        duration: &UnifiedDuration,
         not_busy: bool,
     ) {
-        self.factors[factor as usize].record(id, duration.delays_on_disk_io(false), not_busy);
+        let dur = if factor == InspectFactor::Network {
+            // For network factor, we only care about the raftstore duration.
+            duration.network_duration.unwrap_or_default()
+        } else {
+            // For disk factors, we care about the whole duration.
```

Comment:
NIT: not the whole duration, just the duration on I/O operation => raftstore duration ?

Our inspection focuses specifically on I/O operation costs (measured by `append-log-duration`), intentionally excluding additional latency from `store-wait` duration to isolate pure disk I/O performance, which has been proved that these factors will increase the false-positive rate on the detection of I/O jitters.